pkg/server/routes.go

```diff
@@ -24,9 +24,11 @@ func (s *server) routes(authorizeMiddleware authorizeMiddleware) {
 	systemHandler := handlers.SystemsListHandler{
 		FetchSystems: cedarClient.FetchSystems,
 		Marshal:      marshalFunc,
+		Logger:       s.logger,
```

Comment:
Just as a note: is it going to be long-term maintainable to have all the handlers instantiated in this method? We only have two right now so it's not a problem but I fear this method is going to balloon as we add more.

Reply:
I think this is "the place" to instantiate them. It should be the responsibility of this method. There are a few options for cleaning it up. I think one is separating the API routes to a separate section. The other is looking at dependency injection in the future, where params automatically get filled in. That's the reason we use constructors for the services, eventually you can do something like create a struct with a bunch of handlers and it will fill in all the dependencies by calling the constructors. I think we're a bit off from when this might be a prevalent issue, but the hope is we set it up correctly to slide it in.